src/components/v1/V1Project/modals/ProjectToolDrawerModal.tsx

```diff
@@ -76,6 +83,40 @@ export default function ProjectToolDrawerModal({
     )
   }
 
+  async function setArchived(archived: boolean) {
+    const newMetadata = {
+      ...metadata,
+      archived,
+    }
+    setLoadingArchive(true)
+
+    const uploadedMetadata = await uploadProjectMetadata(newMetadata)
+
+    if (!uploadedMetadata.success) {
+      setLoadingArchive(false)
+      return
+    }
+
+    setUriTx(
+      { cid: uploadedMetadata.cid },
+      {
+        onDone: () => setLoadingArchive(false),
+        onConfirmed: () => {
+          // Create github issue when archive is requested
+          // https://docs.github.com/en/rest/reference/issues#create-an-issue
+          octokit.request('POST /repos/jbx-protocol/juice-interface/issues', {
+            title: `[${archived ? 'ARCHIVE' : 'UNARCHIVE'}] Project: "${
+              metadata?.name
+            }"`,
+            body: `<b>Chain:</b> ${
+              readNetwork.name
+            } \n <b>Handle:</b> ${handle} \n <b>Id:</b> ${projectId?.toString()}`,
```

Comment:
**suggestion**: I imagine this might take markdown, wondering if this works:

```suggestion
            body: `**Chain:** ${readNetwork.name}
            **Project handle:** ${handle}
            **Project ID:** ${projectId?.toString()}`,
```

Reply:
hm... really thought that would work

<img width="463" alt="image" src="https://user-images.githubusercontent.com/79433522/155565446-a2c7ecab-abac-4236-b116-30331003a753.png">

<img width="330" alt="image" src="https://user-images.githubusercontent.com/79433522/155565381-a678e7f7-2353-479b-a26c-026092122a3f.png">

Reply:
LAME. All good haha, thanks for indulging me 😊